format-avro/pom.xml

```diff
@@ -112,8 +112,8 @@
         <artifactId>cdap-maven-plugin</artifactId>
         <configuration combine.self="override">
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline2_2.11[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```

Comment:
these don't need to change. The jars get renamed to cdap-data-pipeline in the corresponding spark folder in the artifacts directory, so they are always in the system as 'cdap-data-pipeline'

Reply:
Got you, reverted